src/components/ActiveWorkspace/ActiveWorkspaceContext.tsx

```diff
@@ -0,0 +1,11 @@
+import {createContext} from 'react';
+
+type ActiveWorkspaceContextType = {
+    activeWorkspaceID?: string;
+    setActiveWorkspaceID: (activeWorkspaceID?: string) => void;
+};
+
+const ActiveWorkspaceContext = createContext<ActiveWorkspaceContextType>({activeWorkspaceID: undefined, setActiveWorkspaceID: () => undefined});
```

Comment:
should we have a default value here?

I usually do `null as ActiveWorkspaceContextType` so that the code throws error if the context is accessed without its corresponding provider up in the component tree.

Reply:
I think that we have some contexts where we define default values like here, for example: `ShowContextMenuContext` and `PopoverContextValue`